hieradata/common.yaml

```diff
@@ -79,6 +79,33 @@ kubernetes::master_nodes:
     - autocrat
     - deadlock
 
+# Kubectl version
+kubectl_package_version: 1.19.4-00
+
+# Kubernetes config (the new setup)
+kube::kubernetes_version: 1.19.4
+# get from https://packages.cloud.google.com/apt/dists/kubernetes-xenial/main/binary-amd64/Packages
+kube::kubernetes_package_version: 1.19.4-00
+# get from https://github.com/kubernetes/k8s.io/
+kube::etcd_version: 3.4.13-0
+# run gen-kube-crt in ocf/utils after editing
+kube::master_nodes:
```

Comment:
this is going to be super nitpicky but the kubernetes project along with everyone else is transitioning away from this terminology

Reply:
I'll change it.